dev/ProgressBar/ProgressBar_themeresources.xaml

```diff
@@ -1,288 +1,43 @@
 ﻿<!-- Copyright (c) Microsoft Corporation. All Rights Reserved. -->
 <ResourceDictionary
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
-    xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)">
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <x:Double x:Key="ProgressBarIndicatorPauseOpacity">0.6</x:Double>
-            <x:Double x:Key="ProgressBarThemeMinHeight">4</x:Double>
             <Thickness x:Key="ProgressBarBorderThemeThickness">0</Thickness>
-            <Color x:Key="SystemControlErrorBackgroundColor">#33FFFFFF</Color>
-            <contract7NotPresent:Color x:Key="SystemErrorTextColor">#FFF000</contract7NotPresent:Color>
+
             <StaticResource x:Key="ProgressBarForeground" ResourceKey="AccentAAFillColorDefaultBrush" />
-            <contract7NotPresent:SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Color="{ThemeResource SystemErrorTextColor}"/>
-            <StaticResource x:Key="ProgressBarBackground" ResourceKey="ControlFillColorTertiaryBrush" />
+            <StaticResource x:Key="ProgressBarBackground" ResourceKey="ControlAAStrokeColorDefault" />
             <StaticResource x:Key="ProgressBarBorderBrush" ResourceKey="ControlStrokeColorDefaultBrush" />
-            <SolidColorBrush x:Key="ProgressBarBackgroundThemeBrush" Color="#59FFFFFF" />
```

Comment:
Removing existing ThemeBrushes is a breaking change if the app is using this resource. Same for changing a resource from brush to color.

Reply:
This definition is copied from generic.xaml, so I think it's ok to remove from WinUI.